manifests/params.pp

```diff
@@ -57,8 +57,38 @@
     $functions_path     = '/etc/rc.d/init.d/functions'
     $start_pidfile_args = '--pidfile $pidfile'
   }
+  
   if $::is_pe == true or $::is_pe == 'true' {
-    # Puppet Enterprise specific settings
+    # < PE 4
+    $is_pe_server      = true
+  }elsif is_function_available('pe_compiling_server_version') {
```

Comment:
Interesting.. so there will be no difference between the FOSS and PE packages and no need to tell the difference from that standpoint? If there is no reason for `is_pe` anymore, why do you bother setting it in this code? Apologies if my comments are getting sidelined from the PR and I appreciate your explanations of the pending fact changes.